app/assets/javascripts/helpers/govspeak-fetch.js

```diff
@@ -0,0 +1,20 @@
+window.GovspeakFetch = { }
+
+window.GovspeakFetch.getBody = function (text) {
+  // Replace last segment to point to the govspeak preview resource path
```

Comment:
Would it be possible to be a little more generic here by just looking at the first two segments of the URL? Otherwise, it's harder to see that this code will work, depending on which pages it occurs on.

Reply:
Sure, will update